openrtb_ext/response.go

```diff
@@ -96,3 +100,33 @@ const (
 	UserSyncIframe UserSyncType = "iframe"
 	UserSyncPixel  UserSyncType = "pixel"
 )
+
+// Bid is Wrapper around original/proxy bid object
+type Bid struct {
+	openrtb2.Bid
+	ID    string `json:"id,omitempty"`    // added omitempty
+	ImpID string `json:"impid,omitempty"` // added omitempty
+
+	OriginalBidCPM float64 `json:"originalbidcpm,omitempty"`
+	OriginalBidCur string  `json:"originalbidcur,omitempty"`
+}
```

Comment:
Why does this embed `openrtb2.Bid` rather than including only the fields we want to represent in our prebid ext?

Reply:
embedded `openrtb2.Bid` with the thought that,
1. `openrtb_ext.Bid` is wrapper around `openrtb2.Bid`
2. It will be easy for adding new fields whenever required just by assigning the values to fields present in `openrtb2.Bid`
3. If we not kept `openrtb2.Bid` object embedded then will need to ensure field names are inline with `openrtb2.Bid` object. Othewise, consumer will need to keep mapping between `openrtb2.Bid` and `openrtb_ext.Bid`. e.g. If consumer wants to log `openrtb2.Bid.SlotInPod` but the field declared under `openrtb_ext.Bid` object was `PodSlot`

Reply:
@ShriprasadM - I pretty strongly disagree with adding the bid object. The issue https://github.com/prebid/prebid-server/issues/2367#issuecomment-1375775940 lays out the fields that are needed in the nonbid. If there are fields of interest to analytics that are missing, please feel to suggest.

Specifically, I will not accept passing `adm` through the nonbid. That field can be many KB large and would be a waste of space. To a lesser concern, the various url fields (burl, nurl, etc) would also a waste of network bandwidth.

Because of those fields, I think it's unwise to pretend that we can "just pass the whole seatbid.bid object".  But I will add it to the agenda for the next meeting. If @SyntaxNode and the committee think it's acceptable to just strip the long-and-useless fields, I will abide.